Emerald Insight.js

```diff
@@ -0,0 +1,240 @@
+				"issue": "3/4",
+				"libraryCatalog": "Emerald Insight",
+				"pages": "192-199",
+				"publicationTitle": "International Journal of Prisoner Health",
+				"url": "https://doi.org/10.1108/IJPH-07-2016-0028",
+				"volume": "13",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.emerald.com/insight/search?q=testing&advanced=true&openAccess=true",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://www.emerald.com/insight/content/doi/10.1108/S1085-462220150000016007/full/html",
+		"items": [
+			{
+				"itemType": "bookSection",
+				"title": "Creating a Cheat-Proof Testing and Learning Environment: A Unique Testing Opportunity for Each Student",
+				"creators": [
+					{
+						"lastName": "Menk",
+						"firstName": "K. Bryan",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Malone",
+						"firstName": "Stephanie",
+						"creatorType": "author"
+					}
+				],
+				"date": "January 1, 2015",
+				"ISBN": "9781784415877 9781784415884",
+				"abstractNote": "AbstractOriginality/value\nThis technique creates opportunities for students to have unique assignments encouraging student to student teaching and can be applied to assignments in any accounting course (undergraduate and graduate). This testing method has been used in Intermediate I and II, Individual Taxation, and Corporate Taxation.",
+				"bookTitle": "Advances in Accounting Education: Teaching and Curriculum Innovations",
+				"extra": "DOI: 10.1108/S1085-462220150000016007",
+				"libraryCatalog": "Emerald Insight",
+				"pages": "133-161",
+				"publisher": "Emerald Group Publishing Limited",
+				"series": "Advances in Accounting Education",
+				"shortTitle": "Creating a Cheat-Proof Testing and Learning Environment",
+				"url": "https://doi.org/10.1108/S1085-462220150000016007",
+				"volume": "16",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
```

Comment:
Can we also add `en` as language for all publications from Emerald?

Reply:
Unfortunately not. Emerald also publishes in other languages (I found German and Spanish at a minimum) and there are no usable language tag on those.

Reply:
Okay, that is pity but it is the publisher's fault we cannot do better.